[EN] Update `Gameplay/Game modifier/Easy` (#14956)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Easy/en.md

|  |  |
| :-- | :-- |
| Acronym | EZ |
| Type | Difficulty Reduction |
| Default shortcut key | `Q` |
| Game modes | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
| Score multiplier | 0.50x |
| Status | Ranked |
| Incompatible mods | [Hard Rock (HR)](/wiki/Gameplay/Game_modifier/Hard_Rock) |

:::

::: alert-note
**Note:** For the [lazer](/wiki/Client/Release_stream/Lazer) version of this article, see: [Easy (lazer mod)](/wiki/Gameplay/Game_modifier/Easy_(lazer))
:::

::: alert-note
**Note:** For other uses, see: [Easy (disambiguation)](/wiki/Disambiguation/Easy)
:::

::: alert-note
**Note:** For the full list of all mods, see: [Game modifier](/wiki/Gameplay/Game_modifier)
:::

The **Easy (EZ)** mod is a difficulty-reducing [game modifier](/wiki/Gameplay/Game_modifier) that halves all difficulty settings of a [beatmap](/wiki/Beatmap).

In all [game modes](/wiki/Game_mode) except [osu!taiko](/wiki/Game_mode/osu!taiko), the Easy mod grants the player two additional lives per play. If the [health bar](/wiki/Client/Interface/Health_bar) drops to zero, the game pauses to remove one life and slowly refills the health bar to approximately 80%.

> The game signals the start of the refill event with the skin's `readysound.wav`. Once the health bar finishes refilling, the `gosound.wav` plays, and the beatmap resumes. If the player runs out of remaining lives and the health bar empties completely, the play fails normally. 

Reviewer: ```suggestion
The game signals the start of the refill event by playing the skin's `readysound.wav` file. Once the health bar finishes refilling, the `gosound.wav` file plays, and the beatmap resumes. If the player runs out of remaining lives and the health bar empties completely, the play fails normally. 
```

### osu!

In [osu!](/wiki/Game_mode/osu!), the Easy mod decreases [circle size (CS)](/wiki/Beatmap/Circle_size), [approach rate (AR)](/wiki/Beatmap/Approach_rate), [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty), and [HP drain (HP)](/wiki/Beatmap/HP_drain_rate) by half.

![EZ gameplay osu!](img/EZ-comparison-osu.jpg "Comparison between a regular play (left) vs a play with the Easy mod enabled (right) in osu!")

> Many players find that the Easy mod does not lower the difficulty of high density beatmaps. The halved approach rate creates a dense, visually cluttered playfield that makes reading the beatmap difficult.

Reviewer: ```suggestion
Many players find that the Easy mod does not lower the difficulty of high-density beatmaps. The halved approach rate creates a dense, visually cluttered playfield that makes reading the beatmap difficult.
```

::: alert-notice
**Notice**
The readability concerns are detailed further in the "[Let's talk Easy Mod](https://osu.ppy.sh/community/forums/topics/56606)" community feature request.
:::

> 

Reviewer: the post is fairly old and looks a bit odd to have in the article at this point. maybe remove?

# Easy (mod)

::: Infobox

<!-- lint ignore heading-increment -->

#### Easy

![Easy mod icon](/wiki/shared/mods/EZ.png)

*![][osu!] Reduces overall difficulty - larger circles, more forgiving HP drain, less accuracy required.*\
*![][osu!taiko] Reduces overall difficulty - notes move slower, less accuracy required.*\
*![][osu!catch] Reduces overall difficulty - larger circles, more forgiving HP drain, less accuracy required.*\

> *![][osu!mania] Reduces overall difficulty - more forgiving HP drain, less accuracy required.*

Reviewer 2: somewhat monstrous, but I guess that's life

one possible solution is remove the description, but it makes more sense to always have it present, because it's the same line for most mods

<img width="1790" height="2414" alt="Image" src="https://github.com/user-attachments/assets/b53794e8-94d9-458f-a375-d8dc1d9413ee" />

Reviewer 3: <img width="1342" height="643" alt="Capture" src="https://github.com/user-attachments/assets/ddc5ef40-edda-43ce-9153-06865c934789" />
I think it'll be fine to leave it here.

Reviewer 2: nah I think let's wait and check when/if this is a widespread issue, and then see if it makes more sense to push descriptions back

Author: Should we put huge descriptions like that under a collapsible section instead? That way, people can just click on it if they want to read it

Reviewer 2: for one, we don't have collapsibles.. the core issue is still that that would take away from the article because it would require more actions to access its contents in full

Author: Oh ok, I forgot we couldn't use HTML tags here. Then yeah let's just leave it for now.

|  |  |
| :-- | :-- |
| Acronym | EZ |
| Type | Difficulty Reduction |
| Default shortcut key | `Q` |
| Game modes | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
| Score multiplier | 0.50x |
| Status | Ranked |
| Incompatible mods | [Hard Rock (HR)](/wiki/Gameplay/Game_modifier/Hard_Rock) |

:::

*For the [lazer](/wiki/Client/Release_stream/Lazer) version of this article, see: [Easy (lazer mod)](/wiki/Gameplay/Game_modifier/Easy_(lazer))*\
*For other uses, see: [Easy (disambiguation)](/wiki/Disambiguation/Easy)*\
*For the full list of all mods, see: [Game modifier](/wiki/Gameplay/Game_modifier)*

> The **Easy** mod, represented by the acronym **EZ**, is a difficulty-reducing [game modifier](/wiki/Gameplay/Game_modifier) that halves all difficulty settings of a [beatmap](/wiki/Beatmap).

Reviewer 2: ```suggestion
The **Easy (EZ)** mod is a difficulty-reducing [game modifier](/wiki/Gameplay/Game_modifier) that halves all difficulty settings of a [beatmap](/wiki/Beatmap).
```

:::

Because players cannot fail mid-song in osu!taiko, the mod does not grant extra lives. Instead, it lowers the health requirement needed to fill the health bar, allowing it to fill much faster than normal.

### osu!catch

In [osu!catch](/wiki/Game_mode/osu!catch), the effects are the same as in osu! with a few parallels and adjustments: fruits fall slower (halved AR), fruits become bigger (halved CS), and two extra lives are granted.

Due to the single-axis movement of the game mode, the larger fruit sizes and slower drop rates can present the same readability challenges found in osu!.

![EZ gameplay catch](img/EZ-comparison-catch.jpg "Comparison between a regular osu! at play (left) vs a play with the Easy mod enabled (right) in osu!catch")

### osu!mania

In [osu!mania](/wiki/Game_mode/osu!mania), the Easy mod reduces overall difficulty (OD) and HP drain by half, and grants two extra lives. Slider velocity and default key counts remain unaffected.

## Trivia

- When watching a replay where an extra life is used, the game skips the health refill animation and continues the play as if the [No Fail](/wiki/Gameplay/Game_modifier/No_Fail) mod were active.

> - In [Multiplayer](/wiki/Client/Interface/Multiplayer) mode, the extra lives mechanic is disabled in [Tag Co-op or Tag Team Vs.](/wiki/Client/Interface/Multiplayer#tag-co-op-/-tag-team-vs) team mode.

Reviewer: ```suggestion
- In [multiplayer](/wiki/Client/Interface/Multiplayer), the extra lives mechanic is disabled in the [Tag Co-op or Tag Team Vs. team modes](/wiki/Client/Interface/Multiplayer#tag-co-op-/-tag-team-vs).
```
